versioned_plugins.rb

```diff
@@ -471,6 +484,23 @@ def write_stack_versions(content, type)
       .gsub("%ECS_VERSION%", @ecs_version)
   end
 
+  PLUGIN_DOCS_LAST_GENERATED_FILE = "plugin_docs_last_generated_time.txt"
+
+  def get_docs_last_generated_file_path
```

Comment:
No need for a method here. I would suggest just having the constant `PLUGIN_DOCS_LAST_GENERATED_FILE` be the full path. 

`PLUGIN_DOCS_LAST_GENERATED_FILE = "#{logstash_docs_path}/plugin_docs_last_generated_time.txt"`